frontend/javascripts/oxalis/model/accessors/tool_accessor.js

```diff
@@ -193,30 +192,30 @@ export function adaptActiveToolToShortcuts(
     activeTool === AnnotationToolEnum.ERASE_BRUSH ||
     activeTool === AnnotationToolEnum.ERASE_TRACE
   ) {
-    // These tools do not have any modifier-related behavior currently.
-    return activeTool;
-  }
-
-  if (activeTool === AnnotationToolEnum.SKELETON) {
-    // The "skeleton" tool is not changed right now (since actions such as moving a node
-    // don't have a dedicated tool). The only exception is "Alt" which switches to the move tool.
-    if (isAltPressed) {
-      return AnnotationToolEnum.MOVE;
+    // These tools do not have any modifier-related behavior currently (except for ALT
+    // which is already handled below)
```

Comment:
Hm, I don't see how ALT is being handled below for the `ERASE_BRUSH` and `ERASE_TRACE` tools. Isn't the ALT handling still being done in the else branch of this condition (so never hit for these two tools)?

Reply:
It's a bit hidden: https://github.com/scalableminds/webknossos/pull/5422/files/ce5deb14bc2a7090152400ea8d0363d698b174e6..1f6249f3e1913c169bf5ce794565f410dc259cf5#diff-6bae9636ea043e9416c5c662d914cbb44ee20ef5eed61c99d2419472e981b072R222

The logic of this method is a bit convoluted. Ideally, this should probably be also delegated to (and unified with) the tool classes. However, it should work at least :see_no_evil: